storage/amazons3/amazons3.go

```diff
@@ -0,0 +1,35 @@
+/*
+ * knoxite
+ *     Copyright (c) 2016, Johannes Fürmann <fuermannj+floss@gmail.com>
```

Comment:
You may want to update the year, here :D

Reply:
Thanks for the hint, fixed.